packages/fxa-auth-server/lib/server.js

```diff
@@ -439,11 +440,16 @@ function metricFactory(statsdClient) {
       path = '/{cors*}';
     }
 
-    statsdClient.timing('url_request', Date.now() - startDate.getTime(), 1, {
-      path: normalizePath(path),
-      method: request.method.toUpperCase(),
-      statusCode,
-    });
+    statsdClient.timing(
+      'url_request',
+      request.info.responded - request.info.received,
```

Comment:
Could this ever be invoked for an error response? As of [Hapi 18](https://github.com/hapijs/hapi/issues/3871), `request.info.responded` is only set for success responses, and set to 0 for error responses, while `request.info.completed` is always set to a timestamp that reflects the processing time. I think we're okay here but not totally sure.

Reply:
I think "when response transmitted successfully" means that the client didn't close the socket before all the of the response was sent, not successful HTTP status.  But since `request.info.completed` is what we really need anyway, I'll switch to that.

Thanks for the great catch!